document-store/src/integrationTest/java/org/hypertrace/core/documentstore/postgres/PostgresDocStoreTest.java

```diff
@@ -162,4 +176,192 @@ public void testDrop() {
     collection.drop();
     Assertions.assertFalse(datastore.listCollections().contains("postgres." + COLLECTION_NAME));
   }
+
+  @Test
+  public void whenBulkUpdatingExistingRecords_thenExpectOnlyRecordsWhoseConditionsMatchToBeUpdated()
```

Comment:
Pl. move this to DocStoreTest to run against both mongo + postgres.
Prior, it was part of MongoDocStoreTest as postgres was not supporting impl for this API. Remove it from `MongoDocStoreTest`

Reply:
Move all the `collection.bulkUpdate` test from `MongoDocStoreTest` and move to `DocStoreTest`